modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/AdminNetworkPolicy/ovn-k-egress-nodes-networks-peer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-egress_{context}"]
+= Egress traffic control for AdminNetworkPolicy and BaselineAdminNetworkPolicy
+
+As a cluster administrator you can use the `AdminNetworkPolicy` (ANP) resource in conjunction with the `BaselineAdminNetworkPolicy` (BANP) resource to secure and limit egress traffic. Using these cluster scoped policies together you can do the following:
+
+* Create egress traffic control for cluster nodes.
+
+* Apply egress traffic control towards Kubernetes API servers.
+
+* Implement egress traffic control towards external destinations.
+
+[id="egress-traffic-control-towards-cluster-nodes_{context}"]
+== Using egress traffic control for ANP and BANP
+
+Using the `matchLabels` and `matchExpressions` fields, cluster administrators can select `nodes` peer and apply egress traffic policy to those nodes.
+
+Cluster administrators can also use CIDR ranges for `networks` peer and apply egress traffic policy to those nodes. The following are benefits to using `nodes` and `networks` peer when blocking access from cluster workloads to specific ports on cluster nodes:
+
+* You do not have to block access to those ports on each external hosts.
+
+* You do not have to manually list the IP address of every node.
+
+* You do not have to change the policy when new nodes are added to the cluster.
+
+The following example allows egress traffic to an API server on port `443` and `6443` by any of the namespaces with a `restricted`, `confidential`, or `internal` level of security and in the `dept: engr` tenancy.
```

Comment:
please check what's running at 6443 on the cluster, I always forget if its openshift-apiserver or kube-api-server and be exact on that piece. Not "AN API" server

Reply:
The endpoints of kube api server listen on 6443 and svc itself listens on 443.
namespace is openshift-kube-apiserver , service is apiserver